jf_agent/main.py

```diff
@@ -2,49 +2,58 @@
 import os
 import sys
 import json
+import urllib3
 
 from jira import JIRA
 from jira.resources import GreenHopperResource
-
 from jf_agent.jira_download import download_users, download_fields, download_resolutions, download_issuetypes, \
     download_issuelinktypes, download_priorities, download_projects_and_versions, download_boards_and_sprints, \
     download_issues, download_worklogs
 
+import stashy
+from jf_agent.bb_download import get_all_users, get_all_projects, get_all_repos, get_default_branch_commits, get_pull_requests
+
+
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('jira_url', help='the fully-qualified base URL for your Jira server, e.g. https://jira.yourcompany.com')
-    parser.add_argument('-o', '--out_dir', nargs='?', default='.', help='output directory')
+    parser.add_argument(
+        '-o',
+        '--out_dir',
+        nargs='?',
+        default='.',
+        help='output directory')
+    parser.add_argument(
+        '--jira-url',
+        type=str,
+        help='the fully-qualified base URL for your Jira server, e.g. https://jira.yourcompany.com',
+        required=False)
+    parser.add_argument(
+        '--bitbucket-url',
+        type=str,
+        help='the fully-qualified base URL for your Bitbucket server, e.g. https://bitbucket.yourcompany.com',
+        required=False)
```

Comment:
I feel like we should support BB cloud, GH cloud, and GHE too.  Fine to start with this one, but might want to think how the configuration works in general.

Reply:
the rationale for cloud providers being, they may still want to audit what info we're getting, and especially if we have a model where we ever read a local repo clone, it'll have to be agent-based.